_includes/setup/instructions/floodgate/viaproxy.md

```diff
@@ -0,0 +1,14 @@
+# Floodgate setup with Geyser for ViaProxy
+
+<div class="alert alert-info" role="alert">
+	To be able to use Floodgate authentication, you need to be able to install Floodgate on the Java server you are connecting to.
+</div>
+
+1. Download and install the platform-specific Floodgate jar (e.g. floodgate-spigot, or floodgate-fabric) [here](https://geysermc.org/download). 
+See the platform setup instructions for a detailed, platform specific setup guide. Then, continue with the instructions below.
+2. Copy the `key.pem` file in the Floodgate config folder to the same directory as Geyser for ViaProxy. 
```

Comment:
This is a bit unclear; can we change this by e.g. adding the path to the directory?